File: w06_java/src/geometry/Circle.java
package geometry;

public class Circle {
    private double radius;

    public Circle(double radius) {
        this.radius = radius;
    }

    public double getRadius() { return radius; }
    public void setRadius(double radius) { this.radius = radius; }

    public double area() {
        return Math.PI * radius * radius;
    }

    public void printInfo() {
        System.out.println("Circle radius=" + radius + ", area=" + area());
    }
}

File: wo6_cpp/Geometry/Circle.cpp
#include <iostream>
#include <cmath>
using namespace std;

namespace Geometry {
    class Circle {
    private:
        double radius;

    public:
        Circle(double r) : radius(r) {}
        double getRadius() const { return radius; }
        void setRadius(double r) { radius = r; }
        double area() const { return M_PI * radius * radius; }

        void printInfo() const {
            cout << "Circle radius=" << radius
                 << ", area=" << area() << endl;
        }
    };
}
